UnityGLTF/Assets/UnityGLTF/Shaders/GLTFStandardInput.cginc

```diff
@@ -154,6 +154,17 @@ half4 SpecularGloss(float2 uv)
 	return sg;
 }
 
+half2 MetallicRough(float2 uv)
```

Comment:
Maybe wrap this definition in an `#if UNITY_VERSION >= 2017.2`? (BTW I don't know what the actual version value is since they changed the naming scheme.)